platform/linux-generic/odp_timer.c

```diff
@@ -1093,6 +1114,11 @@ int odp_timer_capability(odp_timer_clk_src_t clk_src,
 odp_timer_pool_t odp_timer_pool_create(const char *name,
 				       const odp_timer_pool_param_t *param)
 {
+	if (odp_global_ro.init_param.not_used.feat.timer) {
+		ODP_ERR("Trying to use disabled ODP feature.\n");
+		return ODP_TIMER_POOL_INVALID;
```

Comment:
Worth an `ODP_ASSERT()` here? Behavior is undefined if an application tries to use a feature it's said it isn't going to use so these runtime courtesy checks aren't strictly needed, but an assert can be a useful debugging aid.

Reply:
When trying to use a disabled feature the implementation behaviour is undefined and it may crash elsewhere. As `ODP_ASSERT()` won't check anything unless debug mode is enabled and this is not a fast path function I'd prefer to keep this check here.